persist/sqlite/metrics.go

```diff
@@ -66,7 +68,9 @@ func (s *Store) HostMetrics() (result explorer.HostMetrics, err error) {
 			}
 
 			result.TotalStorage += host.Settings.TotalStorage
+			result.TotalStorage += proto4.SectorSize * host.V2Settings.TotalStorage
 			result.RemainingStorage += host.Settings.RemainingStorage
+			result.RemainingStorage += proto4.SectorSize * host.V2Settings.RemainingStorage
 
 			if !host.V2 {
```

Comment:
A small nitpick, but it might be clearer to flip this and frame it with hosts being on V2 as the default. I wouldn’t normally mention it, but it’d be nice if all our new code approached things from that perspective first and foremost.